lib/ui/page/home/page/chat/message_field/view.dart

```diff
@@ -667,11 +668,9 @@ class MessageFieldView extends StatelessWidget {
             Padding(
               padding: const EdgeInsets.symmetric(horizontal: 3),
               child: Text(
-                'label_kb'.l10nfmt({
-                  'amount': e.original.size == null
-                      ? 'dot'.l10n * 3
-                      : e.original.size! ~/ 1024
-                }),
+                e.original.size == null
+                    ? 'dot'.l10n * 3
+                    : e.original.size!.formatFileSize(),
```

Comment:
А нельзя просто 

```suggestion
                e.original.size?.formatFileSize() ?? 'dot'.l10n * 3,
```

?

И название `formatFileSize` - сильно не уверен. В Flutter/Dart'е при конвертации одной величины в другую не пишут `0.formatDoubleNumber()`. Я встречал `toString()`, `parse()`, например, или даже `asBroadcastStream()`. Чем короче, тем лучше. Конкретно в этом случае я бы предложил назвать такое расширением чем-то типа `asFileSize`, возможно.